src/common/interfaces/company-interface.ts

```diff
@@ -33,6 +33,13 @@ export interface Company {
   stop_on_unpaid_recurring: boolean;
 }
 
+export interface CompanyInput {
+  name?: string;
+  subdomain?: string;
+  language_id?: string;
+  currency_id?: string;
```

Comment:
All members are nullable/optional?

Reply:
these will never be nullable, all types are string.

Reply:
> these will never be nullable, all types are string.

fixed